dao/src/main/java/com/carhouse/dao/impl/CarSaleDaoImpl.java

```diff
@@ -68,16 +77,19 @@
      * @param carSaleMapper              mapper to get CarSale object
      * @param parameterSource            class is used to get parameters for sql query
      * @param namedParameterJdbcTemplate for connection with database
+     * @param fileWriter                 to add and update file
      */
     @Autowired
     public CarSaleDaoImpl(final CarFeatureDao carFeatureDao, final CarSaleDtoMapper carSaleDtoMapper,
                           final CarSaleMapper carSaleMapper, final ParameterSource parameterSource,
-                          final NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
+                          final NamedParameterJdbcTemplate namedParameterJdbcTemplate,
+                          final FileWriter fileWriter) {
```

Comment:
think it should be different service.
Why did you decide to place it here instead of separating?